src/config.rs

```diff
@@ -73,6 +79,26 @@ pub(crate) struct CameraConfig {
     #[validate(range(min = 0, max = 31, message = "Invalid channel", code = "channel_id"))]
     #[serde(default = "default_channel_id")]
     pub(crate) channel_id: u8,
+
+    #[cfg(target_os = "linux")]
+    // Maximum number of devices allowed by v4l2loopback (8)
```

Comment:
It doesn't look like this limit is actually a hard limit, since it's possible to create additional devices:

https://github.com/umlaeute/v4l2loopback/issues/367

Can you move this check to runtime, and fail with a nice error message if the user tries to create a device with a number that the kernel turns out not to like?